testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/problems/KnownProblemIds.groovy

```diff
@@ -92,6 +92,9 @@ class KnownProblemIds {
         'dependency-version-catalog:too-many-import-files': ['version catalog error'],
         'dependency-version-catalog:too-many-import-invocation': ['version catalog error'],
         'dependency-version-catalog:no-import-files': ['version catalog error'],
+        'deprecation:generic': ['Generic deprecation'],
```

Comment:
:x: the knowne  ids that come from test coverage are listed at the bottom. See line 165:  // integration test problems

Reply:
Hmm, but this group is used not only for integration tests, but also for production uses.
Following your logic, everything in this file should be an integration test group.